### ISSUE TITLE
[RFE] Support "Stick to fingers"

### ISSUE THREAD

**Issue by BjoernDaase** [mpiannucci/gnome-shell-extended-gestures#71]
First of all, thank you for this great extension, it definitely makes life better when having a touchpad!

Anyway, one thing I would love to see are "Stick to fingers" gestures. E.g. the four finger workspace scrolling implemented by gnome-shell directly has this feature, which makes the gesture feel even more natural. This would be especially interesting for the "Show overview" gesture.  

Unfortunately, I have no clue how complex it would be to implement this. Maybe one could get some ideas/code from this https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/605 or https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/596 gnome-shell MR.

**Comment by mpiannucci** [comment]
Yeah I am not sure how to do this, but it is on my future list of things to try someday. Thanks!

**Comment by BjoernDaase** [comment]
@mpiannucci I actually had a chat with the developer who wrote the Gestures Overhaul (https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/605) and he gives us some perfect starting point when we want to make opening the window overview to follow our fingers: https://gitlab.gnome.org/GNOME/gnome-shell/issues/1338#note_649412.

**Comment by mpiannucci** [comment]
Awesome thanks for that. I will take a look over that
